lib/puppet/application/filebucket.rb

```diff
@@ -148,6 +158,29 @@ def restore
     @client.restore(file, md5)
   end
 
+  def diff
+    checksum_a = args.shift
+    checksum_b = args.shift
```

Comment:
We should probably check for `args.count > 2` and report an error so that users can't do `puppet diff foo bar baz` and have it silently ignore baz.